pages/airdrops/list.tsx

```diff
@@ -20,130 +24,186 @@ interface AirdropListProps {
+                    {getAirdropDate(airdrop.start, airdrop.startType)}
+                  </td>
+                  <td className="p-4">
+                    {getAirdropDate(airdrop.expiration, airdrop.expirationType)}
+                  </td>
+                  <td className={clsx('p-4', { hidden: !wallet.address })}>
+                    <button
+                      className={clsx(
+                        'font-bold text-plumbus uppercase',
+                        'py-1 px-4 rounded border border-plumbus',
+                        'bg-plumbus/10 hover:bg-plumbus/20'
+                      )}
+                      onClick={() => claimOnClick(airdrop.contractAddress)}
+                    >
+                      Claim
+                    </button>
+                  </td>
                 </tr>
-              </thead>
-              <tbody>
-                {airdrops.map((airdrop, idx) => {
-                  return (
-                    <tr key={airdrop.contractAddress} className="hover">
-                      <td>{idx + 1}</td>
-                      <td>
-                        {airdrop.logo && airdrop.logo.url ? (
-                          // eslint-disable-next-line
-                          <img
-                            src={airdrop.logo.url}
-                            alt={airdrop.name}
-                            className="w-12 h-12 rounded-full"
-                          />
-                        ) : (
-                          '-'
-                        )}
-                      </td>
-                      <td>{airdrop.name}</td>
-                      <td>{airdrop.totalAmount}</td>
-                      <td>{airdrop.claimed}</td>
-                      <td>{airdrop.allocation || '-'}</td>
-                      <td>{getDate(airdrop.start, airdrop.startType)}</td>
-                      <td>
-                        {getDate(airdrop.expiration, airdrop.expirationType)}
-                      </td>
-                      <td>
-                        <button
-                          className="p-2 px-6 rounded-lg border"
-                          onClick={() => claimOnClick(airdrop.contractAddress)}
-                        >
-                          CLAIM
-                        </button>
-                      </td>
-                    </tr>
-                  )
-                })}
-              </tbody>
-            </table>
-          ) : (
-            <div className="w-full text-center">No airdrops found!</div>
-          )}
-        </div>
+              ))
+            ) : (
+              <tr>
+                <td colSpan={6} className="p-4 text-center text-white/50">
+                  No airdrops available :(
+                </td>
+              </tr>
+            )}
+          </tbody>
+        </table>
       )}
-      <div className="flex-1"></div>
-    </div>
+    </section>
   )
 }
 
-export default AirdropList
+export default withMetadata(AirdropListPage, { center: false })
```

Comment:
This component is getting quite large. Do you see any opportunities to break it up a bit into multiple components?

Reply:
Sure, I'll break up some parts. Most are still WIP and needed restructuring for #35.

Reply:
Extracted to c4f1476a74ebe976891aedd8654b50a5e7849b4a and ec003a444b567468e23de27b6d37c6d5a2946ad7.